lib/jekyll/tags/include.rb

```diff
@@ -2,14 +2,6 @@
 
 module Jekyll
   module Tags
-    class IncludeTagError < StandardError
-      attr_accessor :path
-
-      def initialize(msg, path)
-        super(msg)
-        @path = path
-      end
-    end
 
```

Comment:
Will this be a breaking change in a scenario where somebody wrote a plugin using this class? (`Jekyll::Tags::IncludeTagError`)